engine/schema/src/main/java/com/cloud/user/AccountVO.java

```diff
@@ -188,7 +188,7 @@ public long getAccountId() {
 
     @Override
     public String toString() {
-        return new StringBuilder("Acct[").append(uuid).append("-").append(accountName).append("]").toString();
+        return String.format("Account {\"id\": %s, \"name\": \"%s\", \"uuid\": \"%s\"}", id, accountName, uuid);
```

Comment:
Though I do agree this is an improvement it might break ecosystem scripts for some. How about
```suggestion
        return String.format("Acct["%s-%s] -- Account {\"id\": %s, \"name\": \"%s\", \"uuid\": \"%s\"}",uuid ,accountName, id, accountName, uuid);
```
to have some more guarantee on backwards compatibility?

Reply:
@DaanHoogland done.